src/server.js

```diff
@@ -137,6 +143,8 @@ function jsonExporter(req, res, next) {
         preview_image_link,
       } = content;
 
+      console.log(blocks);
```

Comment:
@ThomasKindermann Are you adding this intentionally, or is it left over from debugging?